src/test/groovy/hudson/plugins/gradle/AbstractIntegrationTest.groovy

```diff
@@ -16,14 +16,19 @@ import org.jvnet.hudson.test.JenkinsRule
 import spock.lang.Specification
 
 class AbstractIntegrationTest extends Specification {
+
     public final JenkinsRule j = new JenkinsRule()
 
     public final GradleInstallationRule gradleInstallationRule = new GradleInstallationRule(j)
-    final MavenInstallationRule mavenInstallationRule = new MavenInstallationRule(j)
+    public final MavenInstallationRule mavenInstallationRule = new MavenInstallationRule(j)
     public final TestRule noSpaceInTmpDirs = FlagRule.systemProperty("jenkins.test.noSpaceInTmpDirs", "true")
```

Comment:
❓ what's up with these public fields ? Who is needed them ? Shouldn't they be protected, and the `AbstractIntegrationTest ` actually be `abstract` ?

Reply:
Yes, you are right. The only thing is that the hierarchy of these base classes is already pretty messy. I would prefer to leave it like this and then revisit the structure separately.